src/propgrid/editors.cpp

```diff
@@ -1555,7 +1555,7 @@ class wxSimpleCheckBox : public wxControl
         // Due to SetOwnFont stuff necessary for GTK+ 1.2, we need to have this
         wxControl::SetFont( parent->GetFont() );
 
-        SetBoxHeight(12);
+        SetBoxHeight(13);
```

Comment:
Shouldn't this be `wxRendererNative::Get()->GetCheckBoxSize().y`?

Reply:
Maybe. Needs additional checking.

Reply:
Actually it doesn't seem to matter, as it's changed later anyhow, so I'll leave it as is but I think it could be removed completely. But then I also think this entire class could, and should, be removed completely...